lib/pxapi.js

```diff
@@ -152,7 +162,10 @@ function evalByServerCall(ctx, config, callback) {
  */
 function isBadRiskScore(res, ctx, config) {
     if (!res || !pxUtil.verifyDefined(res.score) || !res.action) {
-        ctx.passReason = config.PASS_REASON.INVALID_RESPONSE;
+        ctx.passReason = PassReason.S2S_ERROR;
+        if (!ctx.s2sErrorInfo) {
+            ctx.s2sErrorInfo = new S2SErrorInfo(S2SErrorReason.INVALID_RESPONSE, `Response is ${JSON.stringify(res)}`);         
```

Comment:
res is a huge object usually. Please run it in Debug and make sure you take only relevant information

Reply:
👑